.travis.yml

```diff
@@ -30,8 +54,18 @@ install:
   - conda update -q conda
   # Useful for debugging any issues with conda
   - conda info -a
-  - conda create -q -n test-environment python=$PYTHON pandas=$PANDAS coverage setuptools html5lib lxml pytest pytest-cov
+  - conda create -q -n test-environment python=$PYTHON coverage setuptools html5lib lxml pytest pytest-cov
   - source activate test-environment
+  # Check if we are installing the daily builds for pandas and numpy.
+  # If so, install pandas from conda (to install dependencies) before
+  # installing the correct versions.
+  - if [[ "$PANDAS" == "MASTER" ]]; then
+      conda install pandas;
+      PRE_WHEELS="https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com";
```

Comment:
``pip install numpy pytz python-dateutil`` here, remove the ``conda install pandas`` and this should work.